tests/conftest.py

```diff
@@ -1231,3 +1231,68 @@ def _check_lfs_hashes(repo, expected_file_count=None):
             assert file_count == expected_file_count
 
     return _check_lfs_hashes
+
+
+@pytest.fixture()
+def check_tile_is_reflinked():
+    """
+    Makes sure that a particular tile is reflinked to the same tile in the LFS cache.
+    Makes no asserts at all
+    - on windows, where we don't support reflinks.
+    - if reflink is not supported on the filesystem.
+    - if we lack the tools to check if files are reflinked (clone_checker or )
+    """
+
+    import shutil
+    import subprocess
+
+    import reflink
+
+    from kart import is_windows
+    from kart.lfs_util import get_hash_and_size_of_file, get_local_path_from_lfs_hash
+
+    clone_checker = shutil.which("clone_checker")
+    fienode = shutil.which("fienode")
+    has_checker = bool(clone_checker or fienode)
+
+    def _check_tile_is_reflinked(tile_path, repo, do_raise_skip=False):
```

Comment:
wrt `do_raise_skip`, why _wouldn't_ we always skip if the OS/infra doesn't support the test/feature? That's the pattern we've used elsewhere IIUC

Reply:
I added a couple new tests, both called `test_working_copy_reflink`.
If reflink is missing, we skip them.
However, I also used this same fixture to add some more reflink asserts to some existing import tests. Those tests should now make extra asserts - if they are able to do so - but shouldn't skip the test if they can't.